doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md

```diff
@@ -4,302 +4,241 @@ title: Linux
 permalink: /:path/
 sort_idx: 10
 ---
-
-Use the following instructions to build endpoint SDKs in Java, C, and C++ for Linux.
-
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-
-
-- [Java endpoint SDK](#java-endpoint-sdk)
-- [C endpoint SDK](#c-endpoint-sdk)
-    - [Install build utils and dependencies](#install-build-utils-and-dependencies)
-    - [Compile C endpoint SDK](#compile-c-endpoint-sdk)
-    - [Configure C endpoint SDK](#configure-c-endpoint-sdk)
-- [C++ endpoint SDK](#c-endpoint-sdk-1)
-    - [Install build utils and dependencies](#install-build-utils-and-dependencies)
-    - [Compile C++ endpoint SDK](#compile-c-endpoint-sdk-1)
-    - [Configure C++ endpoint SDK](#configure-c-endpoint-sdk-1)
-- [Quick way to build C/C++ endpoint SDK](#quick-way-to-build-c-c-endpoint-sdk)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+- [Installing prerequisites](#installing-prerequisites)
```

Comment:
Let's use automatic TOC for this purpose:
```
* TOC
{:toc}
```

Sorry, this kind of new style stuff that we are applying ATM:
https://github.com/kaaproject/kaa/pull/757